src/sentry/issues/ingest.py

```diff
@@ -67,6 +73,11 @@ def save_issue_occurrence(
 
 
 def process_occurrence_data(occurrence_data: IssueOccurrenceData) -> None:
+    # Do not hash fingerprints for performance issues because they're already
+    # hashed in save_aggregate_performance
+    # This check should be removed once perf issues are created through the platform
+    if occurrence_data["type"] in get_group_types_by_category(GroupCategory.PERFORMANCE.value):
+        return
```

Comment:
I feel like this should maybe be based on `skip_group_processing`. Otherwise, when we switch over to primarily create via the issue platform we won't hash the fingerprints here.